tests/testthat/test-data-combined.R

```diff
@@ -0,0 +1,741 @@
+# empty initialization ----------------------------
+
+test_that("dataCombined - initialization", {
+  skip_if_not_installed("R6")
+
+  # initialize empty object
+  myCombDat <- DataCombined$new()
+
+  expect_null(myCombDat$dataSets)
+  expect_null(myCombDat$simulationResults)
+  expect_null(myCombDat$groupMap)
+  expect_output(print(myCombDat), "DataCombined:")
+})
+
+# both `DataSet` and `SimulationResults` provided -------------
+
+test_that("dataCombined - both dataSet and SimulationResults provided", {
+  skip_if_not_installed("R6")
+
+  expect_true(R6::is.R6Class(DataCombined))
+
+  # load the simulation
+  sim <- loadSimulation(file.path(getwd(), "..", "data", "MinimalModel.pkml"))
```

Comment:
Check out the file "helper-for-tests" and how it can be used e.g. in https://github.com/Open-Systems-Pharmacology/OSPSuite-R/blob/6021fc93197eb3551009072aff77975606cf1568/tests/testthat/test-simulation.R#L4